civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Module to add functionality for CiviHR Leave and Absences.
+ */
+
+/*
+ * Implements hook_block_info().
+ */
+function civihr_leave_absences_block_info() {
+  $blocks = [];
+  $blocks['my_leave'] = [
+    'info' => t('My Leave'),
+    'cache' => DRUPAL_NO_CACHE,
+  ];
+  return $blocks;
+}
+
+/*
+ * Implements hook_block_view().
```

Comment:
We don't need to define what exactly hook does , only we need to mention hook name according to Drupal standards and practices.